### ISSUE TITLE
Diagram Linking and Navigation

### ISSUE THREAD

**Issue by drnhwly** [archimatetool/archi#202]
I'm new to Archi and new here to the forums, so I apologize if this feature already exists and I haven't found it, or if this isn't the right way to make feature requests...

I'd like to make Views navigable, linking model elements in one diagram to another diagram, to provide a sort of drill-down style navigation. Has anyone considered this yet? Would anyone else find this useful?

**Comment by Phillipus** [comment]
See Issue #123

**Comment by janwilmans** [comment]
Yes, I would love to have this, I currently also use the method used as described by Andy-Turner  in #123 but it making my diagrams look bulky.
